engine/src/execution_graph/logic_controllers/LogicPrimitives.cpp

```diff
@@ -73,5 +78,167 @@ TableViewPair createEmptyTableViewPair(std::vector<cudf::type_id> column_types,
 	return ret;
 }
 
-}
-}
+} // namespace frame
+
+namespace cache{
```

Comment:
its very hard to tell where you made changes if you copy and paste stuff around like this. I am guessing only a few lines below this changed but it look slike all new code.